hubble/client/endpoints.py

```diff
@@ -18,6 +18,7 @@ class EndpointsV2(object):
     upload_artifact: str = 'artifact.upload'
     download_artifact: str = 'artifact.getDownloadUrl'
     delete_artifact: str = 'artifact.delete'
+    delete_multiple_artifacts: str = 'artifact.deleteMany'
```

Comment:
```suggestion
    delete_artifacts: str = 'artifact.deleteMany'
```

Reply:
I prefer long version. `delete_artifact` and `delete_artifacts` are too same.

Reply:
renamed actual method to `delete_multiple_artifacts`.